src/main/js/electron/templates/index.html

```diff
@@ -29,6 +53,8 @@
 <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
 <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"> </script>
 <script src="https://d14fo0winaifog.cloudfront.net/plotly-basic.js"></script>
+<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+
```

Comment:
Does this mean our application is not going to work if we don't have internet connectivity?
Perhaps we should include the libraries in our application...